rasa/nlu/test.py

```diff
@@ -552,83 +548,82 @@ def align_all_entity_predictions(targets, predictions, tokens, extractors):
     :return: list of dictionaries containing the true token labels and token
              labels from the extractors
     """
-
     aligned_predictions = []
-    for ts, ps, tks in zip(targets, predictions, tokens):
-        aligned_predictions.append(align_entity_predictions(ts, ps, tks, extractors))
+    for result in entity_results:
+        aligned_predictions.append(align_entity_predictions(result, extractors))
 
     return aligned_predictions
 
 
-def get_intent_targets(test_data):  # pragma: no cover
-    """Extracts intent targets from the test data."""
-    return [e.get("intent", "") for e in test_data.training_examples]
-
-
-def get_entity_targets(test_data):
-    """Extracts entity targets from the test data."""
-    return [e.get("entities", []) for e in test_data.training_examples]
-
-
 def extract_intent(result):  # pragma: no cover
     """Extracts the intent from a parsing result."""
-    return result.get("intent", {}).get("name")
+
+    intent = result.get("intent", {}) or {}
+    return intent.get("name")
 
 
 def extract_entities(result):  # pragma: no cover
     """Extracts entities from a parsing result."""
     return result.get("entities", [])
 
 
+def extract_tokens(result):  # pragma: no cover
```

Comment:
These single-line extraction functions seem like a bit of a structural deficit, especially when they are only called once.  Could you please remove them?

Same for extracting confidence and intent